### DISCUSSION TITLE
Думаю можно добавить как выбор региона, страны.

### DISCUSSION THREAD

**Discussion by gat3v** [Sokol-Games-GmbH/PixelCombats.PublicApi#142] [Ideas]
**Можете подравить или изменить идею.**

_может быть что игрок с другой страны не знающий другого языка находится на сервере с другими игроками не зная его язык._

_Возможно что это будет хорошое улучшение_

**Comment by kkohno** [comment]
такое частично есть... сервер в новой версии игры будет знать какой у Вас язык... поидее это прокачивать будем да, чтобы все друг-друга в комнате понимали

**Reply by gat3v** [reply]
Ок
